simapp/simd/cmd/root_v2.go

```diff
@@ -69,35 +65,32 @@ func NewRootCmd() *cobra.Command {
 			cmd.SetOut(cmd.OutOrStdout())
 			cmd.SetErr(cmd.ErrOrStderr())
 
-			initClientCtx := *clientCtx
-			initClientCtx = initClientCtx.WithCmdContext(cmd.Context())
-			initClientCtx, err := client.ReadPersistentCommandFlags(initClientCtx, cmd.Flags())
+			clientCtx := *initClientCtx
+			clientCtx = clientCtx.WithCmdContext(cmd.Context())
```

Comment:
In the future we can just do
```go
clientCtx := initClientCtx.WithCmdContext(cmd.Context())
```

Please see https://go.dev/play/p/rA7yAMGaSk6

Reply:
Agree, will be fixed here anyway: https://github.com/cosmos/cosmos-sdk/pull/17913 (this PR was extracting changes from there)

Reply:
Cool and thanks!